akka-bench-jmh/src/main/scala/akka/artery/FlightRecorderBench.scala

```diff
@@ -0,0 +1,62 @@
+package akka.artery
+
+import java.io.File
+import java.nio.channels.FileChannel
+import java.nio.file.StandardOpenOption
+import java.util.concurrent.{CountDownLatch, TimeUnit}
+
+import akka.remote.artery.FlightRecorder
+import org.openjdk.jmh.annotations.{OperationsPerInvocation, _}
+
+@State(Scope.Benchmark)
+@OutputTimeUnit(TimeUnit.MICROSECONDS)
+@BenchmarkMode(Array(Mode.Throughput))
+class FlightRecorderBench {
+
+  @Param(Array("1", "5", "10"))
+  var writers: Int = 0
+
+  val Writes = 10000000
+
+  var file: File = _
+  var fileChannel: FileChannel = _
+  var recorder: FlightRecorder = _
+
+  @Setup
+  def setup():Unit = {
+    file = File.createTempFile("akka-flightrecorder", "dat")
+    file.deleteOnExit()
```

Comment:
Above is an extra safeguard, I also explicitly delete in all test cases. `Teardown` would apply for the whole Spec and I wanted clean journal files for these unit tests to exclude any interference.